src/KafkaFlow/Consumers/OffsetManager.cs

```diff
@@ -20,7 +20,7 @@ internal class OffsetManager : IOffsetManager, IDisposable
                 partition => new PartitionOffsets());
         }
 
-        public void StoreOffset(TopicPartitionOffset offset)
+        public void Commit(TopicPartitionOffset offset)
```

Comment:
I would change this name because we are not committing here. Maybe EnqueueToCommit()